patches/@counterfactual+cf-adjudicator-contracts+0.0.11.patch

```diff
@@ -4,7 +4,7 @@ index d823033..31595b9 100644
 +++ b/node_modules/@counterfactual/cf-adjudicator-contracts/contracts/interfaces/CounterfactualApp.sol
 @@ -1,4 +1,4 @@
 -pragma solidity 0.5.12;
-+pragma solidity ^0.5.13;
```

Comment:
Not sure that this will work since the `.patch` file seems to include commit references. This patch might need to be regenerated. Or maybe not...

Reply:
Seems to work fine for me. I could see that they archived `@counterfactual/cf-adjudicator-contracts` repo and stopped publishing new updates, so we had to patch it like this 🤷‍♂